kano_feedback/WidgetQuestions.py

```diff
@@ -53,39 +54,61 @@ def get_current_prompt_id(self):
         '''
         Returns the id of the current prompt
         '''
-        return self.prompts[self.current_prompt_idx]['id']
+        try:
+            return self.prompts[self.current_prompt_idx]['id']
+        except:
+            return ''
+
 
     def get_current_prompt_type(self):
         '''
         Returns the type of the prompt
         '''
-        if 'type' in self.prompts[self.current_prompt_idx]:
-            return self.prompts[self.current_prompt_idx]['type']
-        else:
-            return "textInput"
+        try:
+            if 'type' in self.prompts[self.current_prompt_idx]:
+                return self.prompts[self.current_prompt_idx]['type']
+            else:
+                return "textInput"
+        except:
+            return ''
 
     def get_current_choices(self):
         '''
         If there are slider options, return them.
         '''
-        if 'choices' in self.prompts[self.current_prompt_idx]:
-            return self.prompts[self.current_prompt_idx]['choices']
+        try:
+            if 'choices' in self.prompts[self.current_prompt_idx]:
+                return self.prompts[self.current_prompt_idx]['choices']
+        except:
+            return ['none']
```

Comment:
I think it might be better to return an empty list here, as that gets feed into the [KanoCombobox](https://github.com/KanoComputing/kano-toolset/blob/jessie/kano/gtk3/kano_combobox.py#L54) in the end.